src/main/java/org/goorm/veri/veribe/domain/book/repository/MemberBookRepository.java

```diff
@@ -1,20 +1,38 @@
 package org.goorm.veri.veribe.domain.book.repository;
 
 import io.lettuce.core.dynamic.annotation.Param;
+import org.goorm.veri.veribe.domain.book.entity.Book;
 import org.goorm.veri.veribe.domain.book.entity.MemberBook;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 
+import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Optional;
 
 public interface MemberBookRepository extends JpaRepository<MemberBook, Long> {
-    List<MemberBook> findAllByMember_Id(Long memberId);
 
     @Query("SELECT mb FROM MemberBook mb " +
             "LEFT JOIN FETCH mb.cards " +
             "JOIN FETCH mb.book " +
             "WHERE mb.id = :memberBookId")
     Optional<MemberBook> findByIdWithCardsAndBook(@Param("memberBookId") Long memberBookId);
 
+    @Query("""
+            SELECT mb.book
+            FROM MemberBook mb
+            WHERE mb.startedAt >= :startOfWeek
+            GROUP BY mb.book
+            ORDER BY COUNT(mb) DESC
+            """)
+    List<Book> findMostPopularBook(LocalDateTime startOfWeek);
+
+    @Query("SELECT mb FROM MemberBook mb WHERE mb.member.id = :memberId ORDER BY mb.createdAt DESC")
+    List<MemberBook> findMemberBookNewest(@Param("memberId") Long memberId);
```

Comment:
```suggestion
    Page<MemberBook> findMemberBookNewest(@Param("memberId") Long memberId, Paegable pageable);
```

지금은 정렬만 되고 페이징은 안되고있는데, 이런 느낌으로 하심 됩니다!  

정렬도 `Pageable pageRequest = PageRequest.of(page, size, sortType.getSort());` 
이렇게 같이 넘겨서 처리할 수도 있어요.  

그런데 JPA에서는 페이지가 0부터 시작해서,  
컨트롤러에서 받아 사용할때, 사용자에게 응답할 때 이부분 보정해주시면 되요